Packs/Netskope/ReleaseNotes/4_0_2.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Netskope (API v2)
+
```

Comment:
change in yml docer image version to :
dockerimage: demisto/python3:3.12.7.117934
The run:
demisto-sdk update-release-notes -i Packs/Netskope/Integrations/NetskopeAPIv2/NetskopeAPIv2.yml